front/pages/poke/[wId]/index.tsx

```diff
@@ -252,6 +252,8 @@ const WorkspacePage = ({
     (ds) => !!ds.connectorProvider
   );
 
+  const FREE_TEST_PLAN_CODE = "FREE_TEST_PLAN";
```

Comment:
Could't load "@app/lib/plans/free_plans.ts" in here cause it needs access to sequelize and a model at the moment. Can refactor immediately if we're not okay with this lazy solution